File: Solicitudes.java
public class Solicitudes extends Ticket{
    private int id;
    private int estado;
    private String descripcion;

    // Getter
    public int getId() {
        return id;
    }

    public int getEstado() {
        return estado;
    }

    public String getDescripcion() {
        return descripcion;
    }

    // Setter
    public void setId(int newId) {
        this.id = newId;
    }

    public void setEstado(int newEstado) {
        this.estado = newEstado;
    }

    public void setDescripcion(String newDescripcion) {
        this.descripcion = newDescripcion;
    }

}

File: src/class/solicitudes.js
import { Ticket } from './ticket.js';

class Solicitudes extends Ticket {
    id;
    estado;
    descripcion;
    constructor(id, estado, descripcion) {
        this.id = id;
        this.estado = estado;
        this.descripcion = descripcion;
    }

    get getId() {
        return this.id;
    }

    set setId(newId) {
        this.id = newId;
    }

    get getEstado() {
        return this.estado;
    }

    set setEstado(newEstado) {
        this.estado = newEstado;
    }

    get getDescripcion() {
        return this.descripcion;
    }

    set setDescripcion(newDescripcion) {
        this.descripcion = newDescripcion;
    }
}